### PR TITLE
Refactor to use cobra, add version command and signal handling

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Fixes https://github.com/giantswarm/prometheus-kemp-exporter/issues/4 and fixes https://github.com/giantswarm/prometheus-kemp-exporter/issues/5

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
@xh3b4sd @teemow 

Good sirs, I present to you this humble Pull Request, that I have toiled so upon, that you may view it, and - if it so pleases you - discern it's worth with a simple review.

File: main.go [hunk 1]
```diff
@@ -1,174 +1,15 @@
...
-
-	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, "")
-	})
-	http.Handle("/metrics", prometheus.Handler())
-
-	http.ListenAndServe(":8000", nil)
+	if err := cmd.RootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(-1)
```
Comment (xh3b4sd) [inline main.go]: Why `-1`?

**Comment by xh3b4sd** [comment]
This looks quiet decent in my point of view.

File: main.go (hunk 1 above)
Comment (JosephSalisbury) [inline main.go]: Huh, that is weird. This is straight from the cobra docs, but I don't like it. I'll change it to `Exit(1)`

**Comment by JosephSalisbury** [comment]
Sir @xh3b4sd , I thank you for graciously taking the time to review my humble request.
I will now perform ye olde merge.

**Pull request merged**